lib/active_model/serializer/adapter/flatten_json.rb

```diff
@@ -6,6 +6,13 @@ def serializable_hash(options = {})
           super
           @result
         end
+
+        private
+
+        # no-op
```

Comment:
What about a more specific comment, nothing big, just to help ppl jumping in the code.